editor/src/core/layout/layout-utils.spec.browser2.tsx

```diff
@@ -72,16 +73,21 @@ describe('maybeSwitchLayoutProps', () => {
     )
     const elementPath = EP.appendNewElementPath(TestScenePath, [NewUID])
 
+    const testSceneProps: ElementProps = {
+      style: {
+        width: 375,
+        height: 812,
+      },
+    }
+
+    const elementProps: ElementProps = {
+      'data-uid': NewUID,
+    }
+
```

Comment:
these don't seem to be used later in the file

Reply:
Removed in 18ae1e3e6.